application/controllers/admin/Database.php

```diff
@@ -369,6 +369,11 @@ protected function actionUpdateSurveyLocaleSettingsGeneralSettings($surveyId)
             'gsid' => $request->getPost('gsid'),
             'template' => $request->getPost('template')
         ];
+        if (empty($input['adminemail']) || empty($input['bounce_email'])) {
```

Comment:
Validation for form inputs is (most of the time :) ) declared inside of the rules function of the model.
In this case see: application/models/Survey.php 
public function rules()
There is already some validation for adminemail and bounce_email using the LSYii_FilterValidator.

As I understand it's still ok to have those fields empty, but when they are not they should be checked for valid email.
You could try with 
array('adminemail', 'LSYii_FilterValidator', 'filter' => 'email', 'skipOnEmpty' => true)